docs/source/api/v3/servicecategories.rst

```diff
@@ -0,0 +1,168 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-service-categories:
+
+**********************
+``service categories``
```

Comment:
This should be the name of the endpoint.

Also, from our API guidelines:

> _"Request paths MUST use 'snake_case' to separate words whenever necessary..."_

So really the route name ought to be `service_categories`.